cli/cli_set_technology.go

```diff
@@ -68,13 +68,11 @@ func (c *cmd) SetTechnology(ctx *cli.Context) error {
 	case internal.CodeSuccess:
 		flag, _ := strconv.ParseBool(resp.Data[0])
 		color.Green(fmt.Sprintf(MsgSetSuccess, "Technology", strings.Join(resp.Data[1:], " ")))
-		c.config.Technology = tech
-		c.config.Obfuscate = false
```

Comment:
The `obfuscate` value is not set to `false` on the daemon side when switching to Nordlynx and because of this, when it is `true` for OpenVPN, switching to Norlynx will break all the commands that use `obfuscated`, e.g. `nordvpn cities`, `countries`